supporting_content/dimensions/pricingunit.md

```diff
@@ -0,0 +1,62 @@
+|:----------|:------------------------------------------------|:-------------------------|
+| AWS       | CUR                                             | pricing/unit             |
+| GCP       | BigQuery Billing Export                         | usage.pricing_unit       |
+| Microsoft | Cost details                                    | UnitOfMeasure            |
+| OCI       | Cost and Usage Report;<br>List Pricing REST API | cost/billingUnitReadable is closest match<br>*Note: The values in this column are similar but not identical to those in the List Pricing REST API; therefore, it may be preferable to rely on the List Pricing REST API, which we already plan to use for ListUnitPrice.* |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider  | Scenario                | Pattern                                                                                                                              |
+|-----------|-------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| AWS       | CUR                     | hours; Hrs; Queries; GB; Secrets; vCPU-Hours; API Requests; Keys; Alarms etc.                                                        |
+| GCP       | BigQuery Billing Export | tebibyte; count; gibibyte; tebibyte; hour; gibibyte hour; gibibyte month; gibibyte hour; gibibyte; hour; count; gibibyte month; etc. |
+| Microsoft | Cost details            | 1 Hour; 10 Hours; 1/Month; 1 GB; 10K; 1 GB-Month; etc.                                                                               |
+| OCI       | Cost and Usage Report   | ONE GiB HOURS STORAGE_SIZE; ONE GiB HOURS MEMORY; etc.                                                                               |
+| OCI       | List Pricing REST API   | Gigabyte Per Hour; etc.<br>*Note: Gigabyte Per Hour corresponds to both ONE GiB HOURS STORAGE_SIZE and ONE GiB HOURS MEMORY*         |
+
+## References and Resources
+
+### AWS
+
+* [Pricing details - AWS Cost and Usage Reports](https://docs.aws.amazon.com/cur/latest/userguide/pricing-columns.html)
+
+### GCP
+
+* [Structure of Detailed data export | Cloud Billing](https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage)
+
+### Microsoft
+
+* [Understand usage details fields - Microsoft Cost Management](https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields)
+
+### OCI
+
+* [Cost and Usage Reports Overview](https://docs.oracle.com/en-us/iaas/Content/Billing/Concepts/usagereportsoverview.htm)
+
+## Discussion / Scratch space
+
+### PricingUnit - non-normalized vs normalized
+
+* After several discussions the team has decided to use **PricingMeasure (non-normalized)** instead of earlier proposed PricingUnit (semi-normalized).
```

Comment:
```suggestion
* After several discussions the team has decided to use **PricingUnit (semi-normalized)** instead of earlier proposed PricingMeasure (non-normalized).
```

Some changes needed below as well to follow new decision.

Reply:
Of course, additional adjustments to the supporting content are coming soon. We tried to communicate that through the commit message. I will let you know when it's ready for review.

Reply:
Doh, that's on me for not expanding the commit message. Thanks @ijurica

Reply:
No problem, I actually don't read them either :). 
I thought you are all resting and wouldn't even notice that I've published a partial change :).

Now, it should be ready for the review.